src/less/event.less

```diff
@@ -3,6 +3,7 @@
 .rbc-event {
   padding: @event-padding;
   background-color: @event-bg;
+  border: 0;
```

Comment:
i think we probably need a more aggressive button style reset if we are gonna use the button tag. e.g. 

```
button {
  padding: 0;
  margin: 0;
  border: none;
  box-shadow: none;
  background: none;
}
```